core/src/test/scala/integration/kafka/admin/RemoteTopicCrudTest.scala

```diff
@@ -301,6 +300,70 @@ class RemoteTopicCrudTest extends IntegrationTestHarness {
     //      "Remote log segments should be deleted only once by the leader")
   }
 
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumName)
+  @ValueSource(strings = Array("zk"))
+  def testClusterWideDisablementOfTieredStorageWithEnabledTieredTopicZK(quorum: String): Unit = {
+    val topicConfig = new Properties()
+    topicConfig.setProperty(TopicConfig.REMOTE_LOG_STORAGE_ENABLE_CONFIG, "true")
+
+    TestUtils.createTopicWithAdmin(createAdminClient(), testTopicName, brokers, numPartitions, brokerCount,
+      topicConfig = topicConfig)
+
+    val tsDisabledProps = TestUtils.createBrokerConfigs(1, zkConnectOrNull).head
+    instanceConfigs = List(KafkaConfig.fromProps(tsDisabledProps))
+
+    assertThrows(classOf[ConfigException], () => recreateBrokers(startup = true))
+  }
+
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumName)
+  @ValueSource(strings = Array("zk"))
+  def testClusterWithoutTieredStorageStartsSuccessfullyIfTopicWithTieringDisabledZK(quorum: String): Unit = {
```

Comment:
```suggestion
  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumName)
  @ValueSource(strings = Array("zk", "kraft"))
  def testClusterWithoutTieredStorageStartsSuccessfullyIfTopicWithTieringDisabled(quorum: String): Unit = {
```
With this, we can have a common test for both the modes.